src/core/solvers/box-fddp.cpp

```diff
@@ -76,21 +76,21 @@ void SolverBoxFDDP::computeGains(const std::size_t& t) {
   }
 }
 
-void SolverBoxFDDP::forwardPass(const double& steplength) {
+void SolverBoxFDDP::forwardPass(double steplength) {
```

Comment:
same here. use `const double`

Even if the `const` might not be useful for built-in types, `const` still improves the readability of the code.